src/aditiGJUN20/PalindromeNumber.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Verify given number is palindrome or not. 
+ * Input: number = 45654
+ * Output : Given number 45654 is palindrome.
+ * 
+ * Input: number = 56789
+ * Output : Given number 56789 is not palindrome.
+*/
+
+package aditiGJUN20;
+
+import java.util.Scanner;
+
+public class PalindromeNumber {
+
+	// Method to check number is palindrome or not.
+	void checkPalindromeNumber(int num) {
+		int reverseNum = 0, inputNum = num;
+		System.out.println("\nInput Number = " + inputNum);
+		while (num > 0) {
+			reverseNum = reverseNum * 10 + (num % 10);
+			num = num / 10;
```

Comment:
we can write assignment operation in while loop. Just FYI.

while((number = number/10)>0)

Reply:
This is new.  i was aware about for loop. But not knowing that for while loop also we can create similar behavior.
Thank you Krishna.